readme.md

```diff
@@ -468,6 +468,7 @@
 - [unhomoglyph](https://github.com/nodeca/unhomoglyph) - Normalize visually similar unicode characters.
 - [i18next](https://github.com/i18next/i18next) - Internationalization framework.
 - [nanoid](https://github.com/ai/nanoid) - Tiny, secure, URL-friendly, unique string ID generator.
+- [stegcloak](https://github.com/kurolabs/stegcloak) - Hide/Watermark text in any unicode string securely - Modern Text Steganography
```

Comment:
You're still not following the formatting guidelines (dot at the end) and I still think the description could be improved. `unicode` => `Unicode`. The text should not be titlecased, etc.